src/main/java/de/retest/recheck/cli/subcommands/Diff.java

```diff
@@ -1,73 +1,169 @@
 package de.retest.recheck.cli.subcommands;
 
-import static de.retest.recheck.printer.DefaultValueFinderProvider.none;
+import static de.retest.recheck.XmlTransformerUtil.getXmlTransformer;
 
 import java.io.File;
 import java.io.IOException;
+import java.nio.file.Path;
 import java.util.List;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import de.retest.recheck.cli.PreCondition;
-import de.retest.recheck.cli.TestReportFormatException;
+import de.retest.recheck.RecheckProperties;
+import de.retest.recheck.SuiteAggregator;
 import de.retest.recheck.cli.utils.ErrorHandler;
 import de.retest.recheck.cli.utils.FilterUtil;
-import de.retest.recheck.cli.utils.TestReportUtil;
-import de.retest.recheck.ignore.CompoundFilter;
+import de.retest.recheck.cli.utils.GoldenMasterUtil;
+import de.retest.recheck.execution.RecheckDifferenceFinder;
 import de.retest.recheck.ignore.Filter;
-import de.retest.recheck.printer.TestReportPrinter;
+import de.retest.recheck.persistence.GoldenMasterProviderImpl;
+import de.retest.recheck.persistence.PersistenceFactory;
+import de.retest.recheck.printer.ActionReplayResultPrinter;
+import de.retest.recheck.report.ActionReplayResult;
+import de.retest.recheck.report.SuiteReplayResult;
+import de.retest.recheck.report.TestReplayResult;
 import de.retest.recheck.report.TestReport;
 import de.retest.recheck.report.TestReportFilter;
+import de.retest.recheck.ui.descriptors.SutState;
+import picocli.CommandLine;
 import picocli.CommandLine.Command;
 import picocli.CommandLine.Option;
 import picocli.CommandLine.Parameters;
 
-@Command( name = "diff", description = "Display differences of given test report." )
+@Command( name = "diff", description = "Display differences of given Golden Masters." )
```

Comment:
```suggestion
@Command( name = "diff", description = "Compare two Golden Masters." )
```

Reply:
Ok, I will change the README.md and test cases accordingly.